src/main/java/com/google/api/common/ApiClock.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2016, Google Inc.
```

Comment:
Hmm I think original copyrights is correct. They are just being moved from another repo.